docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: '3.6'
+networks:
+  # docker network create public
+  public:
+    external: true
+services:
+  frontend:
+    build: ./
+    restart: always
+    ports:
+      - '80'
+    networks:
+      - public
+      - default
+    labels:
+      - 'traefik.enable=true'
+      - 'traefik.backend=talks'
+      - 'traefik.frontend.rule=Host:talks.mindfine.com'
+      - 'traefik.port=80'
+  api:
+    build: ./server/
+    restart: always
+    depends_on:
+      - redis
+    ports:
+      - '3000'
+    networks:
+      - public
+      - default
+    environment:
+      - 'OAUTH_GITHUB_CLIENT_ID=8f16245f33bbfc85e496'
+      - 'OAUTH_GITHUB_CLIENT_SECRET=3faeddb612de511e30c4f9f066d2d00fdc29ca7d'
```

Comment:
Don't think this should be exposed. Better be encrypted with some tools like ansible vault.

Reply:
You are absolutely right on this one... this kind of secret should never been put into arepa. Since in my deployment I’m actually using this file, I was testing it around and accidentally put it here. I may not start using Ansible Vault right away for this lighter deployment, but I’ll at least use configs in server/.env instead.

Reply:
But since in deployment I'm not using server/.env, so yes, it's better to use a vault. Right now I'd like to do it quick and use a docker-compose.override.yml file instead. So just removed the secrets and added notes in README.md on how to override env in a docker-compose.override.yml file. https://github.com/rankun203/test-fullstack/commit/144ea53a4f447da15ae7d376c3b2216fbc1e4cf5 https://github.com/rankun203/test-fullstack/commit/9516255ec5a595926e6f68ad5f50d3eb5d4fbed4

Also the other domains and Traefik related stuff are deployment dependent, so I've removed those as well and put everything into a docker-compose.override.yml file, for now.

Reply:
Resolved: fixed.